### DISCUSSION TITLE
Install vcpkg on Gentoo

### DISCUSSION THREAD

**Discussion by johhnry** [microsoft/vcpkg#29819] [Q&A] [answered]
Hi,

I am trying to install vcpkg on Gentoo. I followed the instructions to bootstrap the installation:

```shell
$ git clone https://github.com/Microsoft/vcpkg.git
$ ./vcpkg/bootstrap-vcpkg.sh -disableMetrics
Downloading vcpkg-glibc...
```

Then nothing... I don't really understand what it does and if it started the build or not.

Can someone help me with that?

Thanks!

- Gentoo: `5.15.80-gentoo-x86_64`
- vcpkg commit: ec96844b8

**Comment by JackBoosY** [comment]
Did the process close after this message?

**Reply by johhnry** [reply]
Yes it just stops with an exit code of `0`:

```shell
$ ./vcpkg/bootstrap-vcpkg.sh -disableMetrics
Downloading vcpkg-glibc...
$ echo $?
0
```

**Reply by JackBoosY** [reply]
Please fill an issue instead of this.

**Reply by dg0yt** [reply]
Perhaps the tool should explicitly print that it completed successful.

**Comment by dg0yt** [comment, marked as answer]
> Then nothing... I don't really understand what it does and if it started the build or not.

This command isn't expected to start a build. In your case, it looks like it completed successful: It downloaded the current  `vcpkg` binary to the `vcpkg` directory. That's all for this step.

**Reply by johhnry** [reply]
My bad I didn't see it! So what's the point of cloning the whole git repository then?

**Reply by dg0yt** [reply]
The repository is the whole collection of recipes and build scripts. Git is the mechanism to update it. And in manifest mode, it enables direct access to specific versions of each recipe.
The tool is update less frequently.
